packages/editor-ui/src/components/NodeCreator/MainPanel.vue

```diff
@@ -11,9 +11,9 @@
 			/>
 			<div class="type-selector">
 				<el-tabs v-model="selectedType" stretch>
-					<el-tab-pane label="All" :name="ALL_NODE_FILTER"></el-tab-pane>
-					<el-tab-pane label="Regular" :name="REGULAR_NODE_FILTER"></el-tab-pane>
-					<el-tab-pane label="Trigger" :name="TRIGGER_NODE_FILTER"></el-tab-pane>
+					<el-tab-pane :label="$translateBaseWithDynamicParam({base: 'mainPanel.filters', param: ALL_NODE_FILTER})" :name="ALL_NODE_FILTER"></el-tab-pane>
+					<el-tab-pane :label="$translateBaseWithDynamicParam({base: 'mainPanel.filters', param: REGULAR_NODE_FILTER})" :name="REGULAR_NODE_FILTER"></el-tab-pane>
+					<el-tab-pane :label="$translateBaseWithDynamicParam({base: 'mainPanel.filters', param: TRIGGER_NODE_FILTER})" :name="TRIGGER_NODE_FILTER"></el-tab-pane>
```

Comment:
`$translateBase` with `interpolate` would be a better fit.

Reply:
Fixed, thanks!